### DISCUSSION TITLE
How to deployed flet app on aapanal

### DISCUSSION THREAD

**Discussion by osama-alshawabkeh** [flet-dev/flet#4329] [Q&A]
### Question

How to deployed flet app on aapanal

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by FeodorFitsner** [comment]
Could you give url to aapanal?

**Reply by osama-alshawabkeh** [reply]
I have on Localhost

**Reply by osama-alshawabkeh** [reply]
I want to know how can I publish Flet application on Localhost aapanal

**Reply by Muddassir-Farooq-official** [reply]
@FeodorFitsner what about cpannel can we host dynamic flet app with pymysql
